ftml/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,64 @@
+on:
```

Comment:
I _think_ any Actions config needs to be in the top-level .github directory

Reply:
Yeah, this just copies all the commits from the repo, there are some things to do afterwards with adjusting CI and getting it integrated into Docker.